File: common/src/main/java/com/rabtman/common/imageloader/ImageConfig.java
package com.rabtman.common.imageloader;

import android.widget.ImageView;

public class ImageConfig {

  protected String url;
  protected ImageView imageView;
  protected int placeholder;
  protected int errorPic;
  protected ImageLoadListener listener;


  public String getUrl() {
    return url;
  }

  public ImageView getImageView() {
    return imageView;
  }

  public int getPlaceholder() {
    return placeholder;
  }

  public int getErrorPic() {
    return errorPic;
  }

  public ImageLoadListener getListener() {
    return listener;
  }
}

File: base-libs/ExImgLoader/src/main/java/com/rabtman/eximgloader/ImageConfig.kt
package com.rabtman.eximgloader

import android.widget.ImageView

open class ImageConfig {
    var url: String? = null
        protected set
    var drawableId = 0
        protected set
    var imageView: ImageView? = null
        protected set
    var placeholder = 0
        protected set
    var errorPic = 0
        protected set
    var listener: ImageLoadListener? = null
        protected set
}